src/runtime/GlobalObjectBuiltinTypedArray.cpp

```diff
@@ -1264,8 +1264,10 @@ static Value builtinTypedArrayJoin(ExecutionState& state, Value thisValue, size_
     ToStringRecursionPreventerItemAutoHolder holder(state, O);
 
     StringBuilder builder;
-    Value elem = O->getIndexedProperty(state, Value(0)).value(state, O);
-    RELEASE_ASSERT(!elem.isUndefinedOrNull());
+    Value elem = String::emptyString;
+    if (!O->asTypedArrayObject()->buffer()->isDetachedBuffer()) {
+        elem = O->getIndexedProperty(state, Value(0)).value(state, O);
+    }
```

Comment:
Ditto. It seems that this part does not match to the spec.

Reply:
True, i updated to the latest standart